src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write your code here
+const logo = document.getElementById('logo');
+
+function boxes(order) {
```

Comment:
Give more meaningful names
```suggestion
function createMessageBox(order) {
```